src/seedu/addressbook/ui/Formatter.java

```diff
@@ -63,4 +64,13 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+    public static String getPrintableString(Printable... printables) {
```

Comment:
Missing header comments.

Ideally you should try to avoid having a similar named static method.